backend/api/Controllers/ReportController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Api.Models;
+using Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Api.Controllers;
+
+[ApiController]
+[Route("reports")]
+public class ReportController : ControllerBase
+{
+    private readonly ILogger<ReportController> _logger;
+    private readonly ReportService _reportService;
+
+    public ReportController(ILogger<ReportController> logger, ReportService reportService)
+    {
+        _logger = logger;
+        _reportService = reportService;
+    }
+
+    /// <summary>
+    /// List all reports on the asset.
+    /// </summary>
+    /// <remarks>
+    /// <para> This query gets all reports (paginated) </para>
+    /// </remarks>
+    [HttpGet]
+    [ProducesResponseType(typeof(IList<Report>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
```

Comment:
This is a get request without any parameters. Can you have a `Bad Request`?

I guess you could add parameters that aren't supposed to be there?

Reply:
Nope, good catch, will fix